hunts/src/tagFilter.js

```diff
@@ -0,0 +1,33 @@
+function getDescendants(tableRow) {
+  const allChildren = [tableRow];
+  for (let i = 0; i < allChildren.length; i++) {
+    const current = allChildren[i];
+    Array.prototype.push.apply(allChildren, current.subRows);
+  }
+  return allChildren;
+}
+
+function isMatchingTagInFilter(tagList, tag) {
+  return tagList.find(x => ['name', 'color', 'id'].every(key => x[key] === tag[key]));
+}
+
+function filterWithTagAdded(tagList, tag) {
```

Comment:
Not sure I understand the naming of this function

Reply:
Ok I see, it's `filter` as a noun, I read it as a verb. I'd also just inline this (and the Removed counterpart).